README.md

```diff
@@ -1,6 +1,6 @@
 # Episcanner Downloader
 
-Episcanner Downloader is a data downloader for the Episcanner application. It retrieves data related to diseases like dengue and chikungunya and saves it in a specified directory.
+Episcanner Downloader is a data downloader for the Episcanner application. It retrieves data related to diseases like dengue, zika and chikungunya and saves it in a specified directory with the formats csv, parquet or duckdb.
```

Comment:
Please replace with the following text:
"Episcanner downloader fetches up-to-date data from the Infodengue databases,  runs the episcanner models and updates the stats that are displayed in the Episcanner application."